ui/src/copyable_text.tsx

```diff
@@ -4,19 +4,29 @@ import React from 'react'
 import { stylesheet } from 'typestyle'
 import { clas, cssVar, pc } from './theme'
 
+const
+  BUTTON_HEIGHT = 24,
+  BUTTON_WIDTH = 34
```

Comment:
This is unnecessary as its only used once anyway and it's clear what it is.

Reply:
You're right, it was useful during my previous implementations.